elements/pfe-textinput/src/pfe-textinput.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base), // needs to be #f0f0f0,
+  BorderColor: pfe-var(surface--border--lighter),
+  BorderBottomColor: pfe-var(surface--border--darker), // TODO: sync on this color #767676,
+  BorderBottomColor--hover: pfe-var(ui-accent),
+  BorderRadius: pfe-var(ui--border-radius),
```

Comment:
Do we need a local variable for border radius?

Reply:
If someone (other than us) wants to make a pill shaped button, they could do that here, right?

Reply:
It would be a pill-shaped input field but yes, they could use this to do that.  Probably better to do that on the theme level though anyway rather than on a field-by-field basis.